index.html

```diff
@@ -536,34 +536,54 @@
-Using emails for cross-context recognition [=purposes=] is therefore not only
-[=inappropriate=] but also threatens key messaging infrastructure. Future iterations of
-the Web platform should ensure that users can log into sites and receive communication
-without relying on email at all.
+A [=person=]'s <dfn>identity</dfn> is the set of characteristics that define them. Their identity
+*in a [=context=]* is the set of characteristics they choose to present to that context. People
+frequently present different identities to different contexts, and also frequently share an identity
+between several contexts.
+
+<dfn>Cross-context recognition</dfn> is the act of recognising that an [=identity=] in one
+[=context=] is the same [=person=] as an [=identity=] in another [=context=]. [=Cross-context
+recognition=] can at times be [=appropriate=] but anyone who does it needs to be careful not to
+apply the [=norms=] of one [=context=] in ways that violate the [=norms=] around use of information
+acquired in a different [=context=]. (For example, if you meet your therapist at a cocktail party,
+you expect them to have rather different discussion topics with you than they usually would, and
+possibly even to pretend they do not know you.) This is particularly true for [=vulnerable=] people
+as recognising them in different [=contexts=] may force their vulnerability into the open.
+
+In computer systems and on the Web, an [=identity=] seen by a particular website is typically
+assigned an <dfn>identifier</dfn> of some type, which makes it easier for an automated system to
+store data about that user.
+
+[=User agents=] support their [=users=]' [=autonomy=] by helping them present their intended
+[=identity=] to each [=context=] that they visit. To do this, [=user agents=] have to make some
+assumptions about the borders between [=contexts=]. By default, [=user agents=] define a
+<dfn>machine-enforceable context</dfn> or <dfn>partition</dfn> as:
+
+* A set of [=environments=] (roughly top-level pages, iframes, and workers)
+* whose [=environment/top-level origins=] are in the [=same site=] (roughly, their [=origin/hosts=]
+  have the same [=host/registrable domain=], but see [[PSL-PROBLEMS]])
+* being visited within the same user agent installation (and profile for user agents that support
+  multiple profiles)
+* between points in time that the user clears that [=site=]'s cookies and other storage
+  (which some user agents do automatically at the end of each session).
+
+Even though this is the default, [=user agents=] are free to both widen and restrict this context as
+their users need. For example, some user agents may help their users present different
+[=identities=] to subdivisions of a single [=site=], while others may help their users present a
+single [=identity=] to a [=site=] across multiple installations or to multiple [=sites=] that the
+user understands represent a single [=party=].
```

Comment:
We could draw lines around what variation is acceptable, but it's not clear to me what those lines should be, so it'll wait for a future PR.